com.netease.CloudMusic.json

```diff
@@ -55,6 +55,22 @@
         ]
     },
     
+    {
+        "name": "keyutils",
```

Comment:
Is this required? If so, please use https://git.kernel.org/pub/scm/linux/kernel/git/dhowells/keyutils.git as source and if it works bump the version to 1.6.3. Also should probably use `LIBDIR` instead of copying and add cleanup the rest.